app/services/proposal_pdf_service.rb

```diff
@@ -381,4 +389,161 @@ def preferred_impossible_dates(field)
   def delatex(string)
     LatexToPdf.escape_latex(string)
   end
+
+  def proposal_organizing_committee
+    @text << "\\section*{\\centering Organizing Committee}\n\n"
+    confirmed_organizer
+    @text << "\\subsection*{A) Early-Career Researcher}\n\n"
+    organizer_early_career
+    @text << "\\subsection*{B) Under represented in STEM}"
+    organizer_represented_stem
+    @text
+  end
+
+  def confirmed_organizer
+    @confirmed_organizers = proposal.invites.where(status: "confirmed",
+                                                   invited_as: "Organizer")
+  end
+
+  def organizer_early_career
+    @confirmed_organizers&.each do |organizer|
+      person = organizer&.person
+      next if person.academic_status.nil?
+
+      @text << "\\noindent #{person.fullname} : #{person.academic_status}\n \n \n"
+    end
+  end
```

Comment:
This does not seem to distinguish Professors Emeritus from early career researchers, i.e. Tenure-track Faculty. It just lists all of the organizers and their academic status. Additionally, it skips those who have an empty academic_status -- and some of them do, due to earlier bugs in our software that saved their response but not their person data.